web/src/styles/decision.css

```
@@ -189,16 +189,38 @@
+/* #1620：ADR 0008 决定 7 多行指引（错误包路径 / 发给作者）
+   消息可收缩+任意断行+区内滚动；按钮不收缩，顶对齐保证窄视口可命中。 */
+.recovery-banner-message {
+  white-space: pre-line;
+  line-height: 1.45;
+  min-width: 0;
+  flex: 1 1 12rem;
+  max-height: min(40vh, 240px);
+  overflow-x: hidden;
+  overflow-y: auto;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)